src/witan/cic/cost.clj

```diff
@@ -0,0 +1,131 @@
+(ns witan.cic.cost
+  (:require [clojure.string :as s]
+            [com.climate.claypoole :as cp]
+            [com.climate.claypoole.lazy :as lazy]
+            [kixi.large :as xl]
+            [kixi.plot.series :as series]
+            [kixi.plot.colors :as colors]
+            [kixi.plot :as plot]
+            [net.cgrand.xforms :as x]
+            [tablecloth.api :as tc]
+            [tech.v3.datatype.functional :as dfn]
+            [tech.v3.dataset :as ds]
+            [tech.v3.dataset.reductions :as ds-reduce]
+            [tick.core :as tick]
+            [tick.alpha.interval :as tai]))
+
+(set! *warn-on-reflection* true)
+
+(defn projection-file->dataset [projection-file-name]
+  (-> projection-file-name
+      (tc/dataset)
+      (tc/rename-columns (fn [s] (-> s (s/lower-case) (s/replace #" " "-") keyword)))))
+
+
+(defn cyp-per-day [{:keys [projection-data
+                           start-date end-date
+                           cpu-pool]}]
+  (let [cyp-per-day-f (fn [ds]
+                        (-> ds
+                            (ds/row-mapcat
+                             (fn [row]
+                               (x/into []
+                                       (map (fn [d] {:date d}))
+                                       #_(comp
```

Comment:
??

Reply:
was a reminder for me in case I messed it up. Removed